lib/accept-version.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const SemVerStore = require('semver-store')
+
+module.exports = {
+  name: 'ACCEPT_VERSION',
+  storage: SemVerStore,
+  deriveVersion: function (req, ctx) {
+    return req.headers['accept-version']
+  }
+}
```

Comment:
I think we should expose and document this as well, it might be possible for a developer to encapsulate this.

Reply:
It's possible, but as for me it would not give any major benefits. If we expose the header which is used to get the version, but keep the same storage - developer will just be able to provide the different header name (or more complex logic?).

If he already does that - why not to craft the whole strategy?

Reply:
That's good!